pkg/tests/end_to_end_tests/query_integration_test.go

```diff
@@ -1111,6 +1209,17 @@ func TestPushdownDelta(t *testing.T) {
 				},
 			},
 		},
+		{
```

Comment:
do we have any end_to_end tests testing the name resolution logic. With and without the schema label?